packages/node/test/manual/release-health/application-mode-session.js

```diff
@@ -0,0 +1,72 @@
+const Sentry = require('../../../dist');
+
+function assertSessions(actual, expected) {
+  if (JSON.stringify(actual) !== JSON.stringify(expected)) {
+    console.error('FAILED: Sessions do not match');
+    process.exit(1);
+  }
+}
+
+function constructStrippedSessionObject(actual) {
+  const { init, status, errors, release, did } = actual;
+  return { init, status, errors, release, did };
+}
+
+let remaining = 2;
+
+class DummyTransport {
+  sendSession(session) {
+    if (remaining === 2) {
+      assertSessions(constructStrippedSessionObject(session),
+        {
+          init: true,
+          status: "ok",
+          errors: 1,
+          release: "2.0.x",
+          did: "ahmed2",
+        }
+      )
+    }
+    else {
+      assertSessions(constructStrippedSessionObject(session),
+        {
+          init: true,
```

Comment:
we should never report `init: true` for the same session more than once

Reply:
Fixed!